index.html

```diff
@@ -3132,40 +3135,44 @@ <h4><code>unsubscribeallevents</code></h4>
         </section>
       </section>
 
+      <!-- Webhook Notifications -->
+
       <section id="sec-http-webhook-profile-protocol-binding-event-listeners">
-        <h4>Event Connections</h4>
+        <h4>Webhook Notifications</h4>
+
+        <p>Webhook notifications are sent by the WebThing to registered Consumers to notify them 
+        of value changes of properties or of events occuring on the WebThing.</p>
 
         <p><span class="rfc2119-assertion" id="http-webhook-profile-protocol-binding-event-connections-1">
-            A HTTP(s) connection to the event listener of a Consumer MUST be initiated and managed by the Web
+            A HTTP(s) connection to the notification listener of a Consumer MUST be initiated and managed by the Web
             Thing.</span>
         </p>
-
+        <p><span class="rfc2119-assertion" id="http-webhook-profile-protocol-binding-event-connections-1a">
+          Notifications must be sent with an <code>HTTP POST</code> request with a 
+          <code>Content-Type</code> header set to <code>application/json.</code>
+        </span>
+        </p>
         <p><span class="rfc2119-assertion" id="http-webhook-profile-protocol-binding-event-connections-2">
-            A Consumer MUST terminate the subscription with the events endpoint of the Web Thing.</span>
+            A Consumer MUST actively terminate the subscription with the (event- or property-) endpoint that is sending notifications
```

Comment:
this sentence is a bit confusing. Do you mean terminate subscription **to** the endpoint?

Reply:
```suggestion
            A Consumer MUST actively terminate the subscription to the (event- or property-) endpoint that is sending notifications
```